test/FunctionalTests/ClientStreamingMethodTests.cs

```diff
@@ -0,0 +1,81 @@
+#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Count;
+using Grpc.AspNetCore.FunctionalTests.Infrastructure;
+using NUnit.Framework;
+
+namespace Grpc.AspNetCore.FunctionalTests
+{
+    [TestFixture]
+    public class ClientStreamingMethodTests : FunctionalTestBase
+    {
+        [Test]
```

Comment:
I was under the impression that `[TestCase]` is the recommended attribute in Nunit3?

Reply:
Hmm seems like `TestCase` is more often used to pass inline arguments to the test method. Although passing no arguments to `TestCase` works, it is functionally the same according to the wiki: https://github.com/nunit/docs/wiki/Attributes. I see both being used in [grpc/grpc](https://github.com/nunit/docs/wiki/Attributes) but we can change the style if you wish.

Reply:
Leaving up to you.

Reply:
Personally I prefer `Test` since it's more concise 😄